schemas/bedrock-authn-token-http.js

```diff
@@ -0,0 +1,171 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const postToken = {
+  title: 'postToken',
+  type: 'object',
+  additionalProperties: false,
+  anyOf: [
+    {required: ['email']},
+    {required: ['account']}
+  ],
+  properties: {
+    email: {
+      type: 'string'
+    },
+    account: {
+      type: 'string'
+    },
+    hash: {
+      type: 'string'
+    },
+    authenticationMethod: {
+      type: 'string'
+    },
+    requiredAuthenticationMethods: {
+      type: 'array'
+    },
+    typeOptions: {
+      type: 'object',
+      properties: {
+        entryStyle: {
+          type: 'string'
+        }
+      }
+    }
+  }
+};
+
+const getTokensQuery = {
+  title: 'getTokensQuery',
+  type: 'object',
+  additionalProperties: false,
+  required: ['email'],
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['password', 'nonce']
+    },
+    email: {
+      type: 'string'
+    }
+  }
+};
+
+const postAuthenticate = {
+  title: 'postAuthenticate',
+  type: 'object',
+  additionalProperties: false,
+  required: ['type'],
+  anyOf: [
+    {required: ['email']},
+    {required: ['account']}
+  ],
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['password', 'nonce', 'totp', 'multifactor']
+    },
+    email: {
+      type: 'string'
+    },
+    account: {
+      type: 'string'
+    },
+    hash: {
+      type: 'string'
+    },
+    challenge: {
+      type: 'string'
+    }
+  }
+};
+
+const postLogin = {
+  title: 'postLogin',
+  type: 'object',
+  additionalProperties: false,
+  anyOf: [
+    {required: ['email']},
+    {required: ['account']}
+  ],
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['password', 'nonce', 'totp', 'multifactor']
+    },
+    email: {
+      type: 'string'
+    },
+    account: {
+      type: 'string'
+    },
+    hash: {
+      type: 'string'
+    },
+    password: {
```

Comment:
Do we really allow password here? Or only `hash`?

Reply:
This test uses password, https://github.com/digitalbazaar/bedrock-authn-token-http/blob/main/test/mocha/10-api.js#L662
Should the test be updated?